tests/core/test_create_dir.py

```diff
@@ -0,0 +1,39 @@
+from sprout.core.create_dir import create_dir
+
+
+def test_create_dir_existing_folder(tmp_path):
+    """Test that the create_dir function checks for existing folder and returns the name
+    if the folder already exists.
+
+    Args:
+        tmp_path: Temporary directory created by pytest
+    """
```

Comment:
```suggestion
    """Checks for an existing folder and returns the name
    if the folder already exists.
    """
```

Don't need the documentation for arguments in tests